distuv/beta.go

```diff
@@ -27,6 +29,17 @@ type Beta struct {
 	Source *rand.Rand
 }
 
+// CDF computes the value of the cumulative density function at x.
+func (b Beta) CDF(x float64) float64 {
+	if x <= 0 {
+		return 0
+	}
+	if x >= 1 {
+		return 1
+	}
+	return mathext.Incbeta(b.Alpha, b.Beta, x)
```

Comment:
My question is a real question. Looking at cephes I think that the factor is included in `Incbeta`. If that's the case, the documentation in `mathext` should be fixed.